REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,351 @@
+# Chart Review Guidelines
+
+Anyone is welcome to review pull requests. Besides our [technical requirements](https://github.com/jenkins-infra/charts/blob/master/CONTRIBUTING.md#technical-requirements) and [best practices](https://github.com/helm/helm-www/tree/master/content/en/docs/chart_best_practices), here's an overview of process and review guidelines.
+
+## Process
+
+The process to get a pull request merged is fairly simple. First, all required tests need to pass and the contributor needs to have a signed [CLA](https://github.com/jenkinsci/infra-cla/blob/master/README.md).
+
+The charts repository uses the OWNERS files to provide merge access. If a chart has an OWNERS file, an approver listed in that file can approve the pull request. If the chart does not have an OWNERS file, an approver in the OWNERS file at the root of the repository can approve the pull request.
+
+To approve the pull request, an approver needs to leave a comment of `/lgtm` on the pull request. Once this is in place some tags (`lgtm` and `approved`) will be added to the pull request and a bot will come along and perform the merge.
+
+Note, if a reviewer who is not an approver in an OWNERS file leaves a comment of `/lgtm` a `lgtm` label will be added but a merge will not happen.
+
+## Immutability
+
+Chart releases must be immutable. Any change to a chart warrants a chart version bump even if it is only changed to the documentation.
```

Comment:
we aren't publishing charts so not sure if we need this? but can adopt it?

Reply:
Not at the moment but I really think we should to be able to install based on a version, multiple time in the past I needed but I never really took the time to setup the workflow for this

Reply:
checkout commit and run?

if we aren't publishing the charts its the exact same workflow isn't it?

Reply:
Yes, the workflow will remain the same except that we will be able to specify different versions for some applications like release.ci.jenkins.io need to be more stable than infra.ci.jenkins.io.